app/views/components/_email-signup-section.html.erb

```diff
@@ -0,0 +1,30 @@
+<%
+  local_assigns[:heading_level] ||= 3
+  local_assigns[:margin_bottom] ||= 0
+  link_text ||= false
+  link_href ||= false
+  intro ||= false
```

Comment:
Should this variable be called `heading`?

Reply:
I think I called it `intro` because (for some reason) that's what it's named in the content item. Renamed to `heading` as it makes more sense.